WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -36,6 +38,7 @@ private async Task<InputRegistrationResponse> RegisterInputCoreAsync(InputRegist
 
 		using (await AsyncLock.LockAsync(cancellationToken).ConfigureAwait(false))
 		{
+			var beforeInside = DateTimeOffset.UtcNow;
```

Comment:
```suggestion
			var before = DateTimeOffset.UtcNow;
```

Local variable, why specify it's inside a function? Same question about the other `beforeInside`s.

Reply:
I named the variable like this. I prefer to keep that - not confusing IMO.